modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,212 @@
+----
+
+.. Identify the {product-title} cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create and manage AWS Identity and Access Management (IAM) roles and policies:
++
+.. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC) and the AWS ALBO, which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/main/assets/iam-policy.json[here].
++
+.. Create an AWS IAM role and assign the AWS IAM policy displayed in step 2a to that role. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+.. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
++
+--
+[source,json]
+----
+"{
+    ""Version"": ""2012-10-17"",
+    ""Statement"": [
+        {
+            ""Effect"": ""Allow"",
+            ""Principal"": {
+                ""Federated"": ""arn:aws:iam::{AWS_AccountNo}: <1>
+
+                {Cluster_OIDC_Endpoint}"" <2>
+            },
+            ""Action"": ""sts:AssumeRoleWithWebIdentity"",
+            ""Condition"": {
+                ""StringEquals"": {
+                    ""{Cluster's OIDC Endpoint}:sub"": [
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager"",
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster""
+                    ]
+                }
+            }
+        }
+    ]
+}"
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number.
+<2> Replace '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.
+
+[NOTE]
+====
+Do not include the 'https' portion of the OIDC DNS url when replacing '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.  Only the alphanumeric information that proceeds the '.net/' within the URL is needed.
+====
+--
+. Add the tags necessary for subnet discovery:
+
+.. Add the following {Key: Value} tag to the VPC hosting the {product-title}  cluster, replacing '{Cluster Infra ID}' with the Infra ID specified in step 1a:
+
+
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+
+.. Add the following ELB V2 {Key: Value} tags to the private subnets and (optional) public subnets:
+
+* Private subnets: kubernetes.io/role/internal-elb:1
+* Public subnets: kubernetes.io/role/elb:1
++
+[NOTE]
```

Comment:
Covers line 21 in https://docs.google.com/spreadsheets/d/1nwUdersMAaKZ4AUX_KhCpgsjL6DByxVy2B95CNqaeYY/edit#gid=1241421272